app/client/cli/keys.go

```diff
@@ -0,0 +1,465 @@
+				}
+
+				fmt.Printf("Key created: %s\n", kp.GetAddressString())
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysUpdateCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Update <addrHex>",
+			Short:   "Update <addrHex>",
+			Long:    "Updates the passphrase of <addrHex> in the keybase",
+			Aliases: []string{"update"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				pwd = readPassphrase(pwd)
+
+				newPwd = readPassphraseMessage(newPwd, "New passphrase: ")
+
+				err = kb.UpdatePassphrase(addrHex, pwd, newPwd, hint)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Updated key: %s\n", addrHex)
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysDeleteCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Delete <addrHex>",
+			Short:   "Delete <addrHex>",
+			Long:    "Deletes <addrHex> from the keybase",
+			Aliases: []string{"delete"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				pwd = readPassphrase(pwd)
+
+				err = kb.Delete(addrHex, pwd)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Key deleted: %s\n", addrHex)
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysGetCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "List",
+			Short:   "List all keys",
+			Long:    "List all of the hex addresses of the keys stored in the keybase",
+			Aliases: []string{"list"},
+			Args:    cobra.ExactArgs(0),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				addresses, _, err := kb.GetAll()
+				if err != nil {
+					return err
+				}
+
+				for _, addr := range addresses {
+					fmt.Println(addr)
+				}
+
+				return kb.Stop()
+			},
+		},
+		{
+			Use:     "Get <addrHex>",
+			Short:   "Get <addrHex>",
+			Long:    "Get the keypair <addrHex> from the keybase",
```

Comment:
This just prints the PublicKey, so we should update the description.

You're using it similar to an `Exists`, right?